app/src/store/slices/deploy-slice.ts

```diff
@@ -87,6 +87,8 @@ export const fetchDeploys = createAsyncThunk(
         pageNum,
       });
 
+      console.log({ deploys });
```

Comment:
Do we still need this?

Reply:
Oops! Removed locally but forgot to push 😅